File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/inputs/inputWithDefaultValueForObject/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.inputs.inputWithDefaultValueForObject.expected;

import java.lang.String;

public class DgsConstants {
  public static class PERSON {
    public static final String TYPE_NAME = "Person";

    public static final String Name = "name";

    public static final String Age = "age";

    public static final String Car = "car";
  }

  public static class CAR {
    public static final String TYPE_NAME = "Car";

    public static final String Brand = "brand";
  }

  public static class MOVIEFILTER {
    public static final String TYPE_NAME = "MovieFilter";

    public static final String Director = "director";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/inputWithDefaultValueForObject/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.inputWithDefaultValueForObject.expected

import kotlin.String

@Generated
public object DgsConstants {
  @Generated
  public object PERSON {
    public const val TYPE_NAME: String = "Person"

    public const val Name: String = "name"

    public const val Age: String = "age"

    public const val Car: String = "car"
  }

  @Generated
  public object CAR {
    public const val TYPE_NAME: String = "Car"

    public const val Brand: String = "brand"
  }

  @Generated
  public object MOVIEFILTER {
    public const val TYPE_NAME: String = "MovieFilter"

    public const val Director: String = "director"
  }
}
